src/xdocs/writingchecks.xml

```diff
@@ -581,14 +581,14 @@ public class MethodLimitCheck extends AbstractCheck
       </p>
 
       <p>For Linux/Unix OS:</p>
-      <source>
+      <source class="wrap-content">
 java -classpath mycompanychecks.jar:checkstyle-${projectVersion}-all.jar \
```

Comment:
please remove from source " "s and combine in one line, ones we have wrapping we could keep it as one line.
Please do the same for all other cases with " \" in this file.